src/components/faq/helpers/utils.ts

```diff
@@ -0,0 +1,29 @@
+export function removeHtmlTags(html: string): string {
+  return html.replace(/<[^>]+>/g, '')
+}
+
+/**
+ * Traverse any props.children to get their combined text content.
+ *
+ * Note: This does not add whitespace for readability: `<p>Hello <em>world</em>!</p>`
+ * yields `Hello world!` as expected, but `<p>Hello</p><p>world</p>` returns
+ * `Helloworld`, just like https://mdn.io/Node/textContent does.
+ *
+ * Source: https://stackoverflow.com/a/63173682/1265111
+ */
+export function getTextContentFromJSX(elem: React.ReactElement | string): string {
+  if (!elem) {
+    return ''
+  }
+
+  if (typeof elem === 'string') {
+    return elem
+  }
+
+  const children = elem.props && elem.props.children
+  if (children instanceof Array) {
+    return children.map(getTextContentFromJSX).join('')
+  }
+
+  return getTextContentFromJSX(children)
+}
```

Comment:
In order to avoid checking JSX components for text content we can replace the actual content definition from:

```tsx
  {
    header: 'Какви суми могат да се даряват? Има ли минимална и максимална сума?',
    content: (
      <ContentTypography>
        Сумите, които можете да дарите към момента са: 2 лв., 5 лв., 10 лв., 20 лв., 50 лв., 100 лв.
        или сбор от няколко от изброените суми. Така например, ако искате да дарите 200 лв, ще
        можете да направите 2 дарения по 100 лв. Ще се постараем в най-кратки срокове да намерим
        технологични решения, които да дадат повече свобода за дарителите ни в избора на сума.
        Отделно има и опция за регулярни дарения към целогодишните кампании.
      </ContentTypography>
    ),
  },
```

to 

```tsx
  {
    header: 'Какви суми могат да се даряват? Има ли минимална и максимална сума?',
    content: "Сумите, които можете да дарите към момента са: 2 лв., 5 лв., 10 лв., 20 лв., 50 лв., 100 лв. или сбор от няколко от изброените суми. Така например, ако искате да дарите 200 лв, ще можете да направите 2 дарения по 100 лв. Ще се постараем в най-кратки срокове да намерим технологични решения, които да дадат повече свобода за дарителите ни в избора на сума.
        Отделно има и опция за регулярни дарения към целогодишните кампании."
  },
```

Reply:
There are a few questions which are made of a bit more complex markup with lists/links:
```
<ContentTypography>
  Разбира се!. Маркетинговата стратегия на Подкрепи.бг е поместена{' '}
  <ExternalLink variant="subtitle1" href={undefined}>
    {' тук.'}
  </ExternalLink>
</ContentTypography>

<ContentTypography>
  Ние сме общност от доброволци, обединени от идеята да създаваме устойчиви решения за
  развитието на дарителството в България. Нашият основен проект е разработването и
  реализацията на сигурна, дигитална платформа с отворен код за набиране на дарения. Вярваме,
  че това е начинът да:
  <List>
    <ListItem>- помогнем на повече нуждаещи се.</ListItem>
    <ListItem>- съкратим времето за обработка на кампаниите.</ListItem>
    <ListItem>- опростим управлението им.</ListItem>
    <ListItem>
      - дадем възможност за едновременна работа по голям брой кампании и предоставим по-лесен
      достъп за осъществяване на контрол над работата на платформата.
    </ListItem>
  </List>
  Ние предлагаме различен модел на управление, базиран на безкомпромисна прозрачност,
  самоиздръжка без комисиона от кампаниите и достойно заплащане на бъдещите служители на
  организацията. Принципи, в които вярваме и сме твърдо решени да следваме неотклонно!
</ContentTypography>
```

Reply:
@valkirilov Alright, I get the point of why this component is structured in such a way.

Let's leave it as is for the time being. This will help us move on with the current state and have the optimization in mind for the future.